wisl/lib/ParserAndCompiler/wisl2Gil.mli

```diff
@@ -1,3 +1,3 @@
 open Gillian.Gil_syntax
 
-val compile : WProg.t -> (Annot.t, string) Prog.t
+val compile : string -> WProg.t -> (Annot.t, string) Prog.t
```

Comment:
can you make that `compile : filename:string -> WProg.t -> (Annot.t, string) Prog.t` ?
I like to use tagged parameters when it has a common type like  `string` and it's impossible to know what it's used for from the signature. Here, the string parameter is very non-informative :/

If you can make the same change for C that would be amazing